pipelines/performance-tests-pipeline.yml

```diff
@@ -85,14 +18,6 @@ resources:
     uri: git@github.com:ONSdigital/census-rm-deploy.git
     private_key: ((github.service_account_private_key))
 
-- name: census-rm-kubernetes-release
-  type: git
-  source:
-    uri: git@github.com:ONSdigital/census-rm-kubernetes.git
-    private_key: ((github.service_account_private_key))
-    paths: [release/*]
-    branch: master
```

Comment:
We still need this resource, the deployment jobs are broken if they don't `get` it.

Reply:
Added back in